en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,90 @@
+Customize Content Attribute Display
+===================================
+
+Content attributes type are mixed, it's difficult to render them to user. When we
+want to display a string in datatable or in other place, value give array or object.
+To avoid the problem, Open Orchestra add a variable string in HTML Format to
+content attributes. Before to save the attribute, Open Orchestra use a
+``ValueTransformerManager`` to convert value into string.
+
+Manager
+-------
+
+Manager get all ``ValueTransformerInterface`` with
+``open_orchestra_backoffice.value_transformer.strategy`` tag.
+To generate the string value, manager will search in this transformer list to find the
+first one supporting the current fieldType and value.
+Then it will use the transformer method to generate the string value from original value.
+If any transformer support modification, string value take directly the original value in argument.
+
+Transformers
+------------
+
+Adding a transformer
+~~~~~~~~~~~~~~~~~~~~
+
+To adding a new transformer, create a new service which implements
+``OpenOrchestra\BackofficeBundle\EventSubscriber\DataTransformer\ValueTransformerInterface``
+with the tag ``open_orchestra_backoffice.value_transformer.strategy``.
+It will be add to ``ValueTransformerManager`` :
+
+.. code-block:: yml
+
+    open_orchestra_backoffice.value_transformer.array:
+            class: %open_orchestra_backoffice.value_transformer.array.class%
+            tags:
+                - { name: open_orchestra_backoffice.value_transformer.strategy }
+
+Then implements ``ValueTransformerInterface`` in your transformer :
+
+- ``transform`` : transform value to an Html string
+- ``support`` : return true if the transformer must be used
+- ``getName`` : use by manager as index for the transformer
+
+.. code-block:: php
+
+    class ArrayToHtmlStringTransformer implements ValueTransformerInterface
+    {
+        public function transform($data)
+        {
+            //transform data to html
+
+            return $htmlData
+        }
+
+        public function support($fieldType, $value)
+        {
+            return gettype($value) == 'array' && $fieldType == 'choice';
+        }
+
+        public function getName()
+        {
+            return 'array';
+        }
+    }
+
+Override a transformer
+~~~~~~~~~~~~~~~~~~~~~~
+
+To override a transformer, add a new transformer with the same ``getName`` method and
+add this new one to the manager service.
+The last one call will override the others.
+
+.. code-block:: php
+
+    <?php
+
+    namespace MyBundle\DependencyInjection\Compiler;
+
+    use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+    use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+    class OverrideValueTransformerCompilerPass implements CompilerPassInterface
```

Comment:
if you want to override a transformer, there is no need for you to add a compiling method, this will be done in the one you have written as long as the service is tagged